theme/theme.go

```diff
@@ -545,6 +545,36 @@ func loadCustomFont(env, variant string, fallback fyne.Resource) fyne.Resource {
 	return res
 }
 
+func safeColorLookup(n fyne.ThemeColorName, v fyne.ThemeVariant) color.Color {
+	col := current().Color(n, v)
+	if col == nil {
+		return &color.NRGBA{}
```

Comment:
Would this be black? We may want to default to an outrageous color (magenta?) to draw the dev's attention to a missing definition

Reply:
It is transparent, that felt right. When we add new colours this could happen.
Should we indicate that clearly or fall back to no color?

(note this is not normal fallback, just if they did not build their theme completely)

Reply:
I'm just thinking that if a dev built an empty theme, nothing would be visible

Reply:
Is nothing visible better or worse than a bug ugly coloured box?

I guess it's the intermediate I am more concerned about - as the toolkit should move on the flexible theme API was designed with the idea that a theme code could lag without significant ramifications. That's why I think transparent makes a better fallback in this case.

Ideally they would delegate to `theme.DefaultTheme().lookuptype(...)` but if they have chosen not to that is where this code gets called.

Reply:
I'm not sure what is better, any designers care to chime in?

Whatever the result, it should be publicly documented, and maybe we should return a constant rather than a fresh allocation, ie. theme.ColorFallback or color.Transparent